### PR TITLE
TASK: Update nodemigration documentation

### PR DESCRIPTION
**Pull request by dlubitz**
Updates the nodemigration documentation to the new filters and transformations.

### CODE REVIEW

File: Neos.Neos/Documentation/References/NodeMigrations.rst [hunk 1]
```diff
@@ -199,11 +196,27 @@ Options Reference:
...
+
+Moves a dimension space point globally.
+
+``from`` (array)
+  Source Dimension Space Point as array. E.g. ["language" => "es", "country" => "es"]
+``to`` (array)
+  Target Dimension Space Point as array. E.g. ["language" => "es", "country" => "ar"]
+
+
 RemoveNode
```
Comment (ahaeslich) [inline Neos.Neos/Documentation/References/NodeMigrations.rst:210]: Why isn't this one called `RemoveNodeAggregate`? `RenameNode` is now called `RenameNodeAggregate`.

Comment (dlubitz) [reply Neos.Neos/Documentation/References/NodeMigrations.rst:210]: IDK! We can discuss that, but this just documents the current state.

**Review by kdambekalns — APPROVED** [review]
👍 by 👀

**Review by kitsunet — APPROVED** [review]
Fine for me for now just fixing docs.

File: Neos.Neos/Documentation/References/NodeMigrations.rst (hunk 1 above)
Comment (kitsunet) [reply Neos.Neos/Documentation/References/NodeMigrations.rst:210]: might need another issue / task then :)

**Pull request merged**